### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by haftamudesta**
This is the leaderboard project setup. The leaderboard website displays scores submitted by different players. It also allows you to submit your score.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+### Key Features <a name="key-features"></a>
+
+- **Add Lists**
+- **Remove Completed Lists**
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```
Comment (t-yanick) [inline README.md]: - [ ] To make our ReadMe professional we should avoid having empty links or empty links. If you don't yet have a demo link for your project, you can using the text coming soon or something. Your link is giving error 404 which is good.
Screen Shot 2023-02-07 at 2 51 38 AM

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                placeholder="Search..."
+                id="search-text-input"
+                class="curved-input"
+              /><i class="fa-solid fa-magnifying-glass"></i>
+            </div>
+          </div>
+        </div>
+      </header>
+      <h2 class="leader">Leaderboard</h2>
+      <div class="container">
```
Comment (t-yanick) [inline src/index.html]: - [ ] Example case for using flex as a class: `<div class="container flex">`
Create a flex class now in your CSS.
Check every element in your html that you want it to be display flex and pass flex class to them

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+.footer {
+  width: 80%;
+  margin-left: 10%;
+  border: 4px solid black;
+  background: rgb(180, 171, 171);
+}
+
+.footer p {
+  display: flex;
```
Comment (t-yanick) [inline src/style.css]: - [ ] Great job working on this milestone. We have to try as much as possible to follow the [DRY principles.](https://vanseodesign.com/css/dry-principles/). When I look at your CSS, I find so many `display: flex` . You can use this just once in your CSS and pass flex in your html as a class like this `<div class="container flex">`
```
.flex{
   display: flex;
       }
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @haftamudesta , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Excellent use of Gitflow
:heavy_check_mark: Basic project structure well respected

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Making your `PR title `more descriptive and also the `body` of the PR.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @haftamudesta  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional README file :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**